cmake/ONNXConfig.cmake.in

```diff
@@ -6,10 +6,7 @@
 # library version information
 set(ONNX_VERSION "@ONNX_VERSION@")
 
-if(@Protobuf_FOUND@)
-  list(APPEND CMAKE_MODULE_PATH "@PROTOBUF_DIR@")
-endif()
-find_package(Protobuf REQUIRED)
+find_package(Protobuf REQUIRED CONFIG)
```

Comment:
If ONNX links static protobuf, then there is no need to find protobuf

Reply:
If ONNX itself is static you still need both, no?

Reply:
I mean, if ONNX links static protobuf libraries, we don't need to find protobuf on user's host.

Reply:
Static dependencies are not transitive, so you would need both protobuf and ONNX in the case ONNX is also static. I can add a check which takes that into account, indeed...